src/amb_kf.c

```diff
@@ -35,6 +35,10 @@
  * The expectation value of carrier + code / lambda is
  * integer ambiguity + bias. Regardless of bias, this is an
  * important measurement.
+ *
+ * \param carrier A carrier phase measurement in units of wavelengths.
+ * \param code    A code measurement in the same units as GPS_L1_LAMBDA_NO_VAC.
+ * \return A rough estimate of the integer ambiguity.
```

Comment:
rough here sounds like imprecise language, perhaps consider dropping that and detailing the limitations of this measurement technique above?